src/index.html

```diff
@@ -14,7 +14,7 @@
     <title>Coso</title>
   </head>
   <body>
-    <section class="container navigation">
+    <section class="navigation">
       <img src="/coso/app/vendors/images/logo-black.svg" alt="logo" />
```

Comment:
@angelicazgr this paths should be relative to this file, for example

`../vendors/images/logo-black.svg`